data/schemas/keyboard.jsonschema

```diff
@@ -33,6 +33,14 @@
                 "voices": {"type": "boolean"}
             }
         },
+        "autocorrect": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "enabled": {"type": "boolean"},
+                "data_h": {"type": "string"}
```

Comment:
```suggestion
                "enabled": {"type": "boolean"}
```

Reply:
> `info.json` having context of headers bleeds implementation details into something that should be fairly generic. Its also questionable if the feature should be enabled at the keyboard level, which makes the schema/mappings changes not required. My view would be that it should never be.
> 
> Providing a default header to fall back on, and then warning seems a tad strange. I'm not really sure what use case that is trying to cover (especially if you take the viewpoint above).

So perhaps we go with:
```c
#if __has_include("autocorrect_data.h")
#    include "autocorrect_data.h"
#else
#    include "autocorrect_data_default.h"
#endif
```
......and just leave it up to the keymap to have it present?